### PR TITLE
Added themes

### LINKED ISSUE #419: To add a variety of themes like in the chat app
**Actual Behaviour**

Only light and dark themes available

**Expected Behaviour**

To add more themes

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #419 

Changes: Added feature to add custom theme in the background

Surge Deployment Link: https://pr-448-fossasia-susi-sill-cms.surge.sh

Screenshots for the change:
screen shot 2018-05-12 at 5 23 53 pm

Next Steps
* Adding custom colors for other components
* Try to sync it with server

### CODE REVIEW

File: src/components/Settings/Settings.js [hunk 1]
```diff
@@ -146,6 +175,23 @@ class Settings extends React.Component {
                 labelColor='#fff'
                 onClick={this.handleSubmit}
               />
+              <h3 style={head}>OR</h3>
+              <div>
+              <div>
+        <RaisedButton label="Custom Themes" onClick={this.handleOpen} />
```
Comment (akshatnitd) [inline src/components/Settings/Settings.js]: Fix indentation

File: src/components/CustomThemes/colors.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.buttonStyle {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    padding: 2px 2px 2px 2px;
+    margin: 3px 2px;
+}
```
Comment (akshatnitd) [inline src/components/CustomThemes/colors.css]: Add a EOF

File: src/components/Settings/Settings.js [hunk 3]
```diff
@@ -106,21 +108,48 @@ class Settings extends React.Component {
...
         };
+        const actions = [
+            <FlatButton
+              label="Reset"
+              primary={true}
+              onClick={this.handleCloseReset}
+              key="1"
+            />,
+            <FlatButton
+              label="Submit"
```
Comment (akshatnitd) [inline src/components/Settings/Settings.js]: Can be changed to "Apply"

**Comment by stealthanthrax** [comment]
@Akshat-Jain @akshatnitd @AnupKumarPanwar , done 🤘

**Comment by anshumanv** [comment]
Though IMO what we're changing is not theme exactly, it'sbackground color, don't you think? @stealthanthrax :thinking:

**Comment by anshumanv** [comment]
Okay it's mentioned, cool :+1: :smile:

**Comment by stealthanthrax** [comment]
@dynamitechetan @madhavrathi @saurabhjn76 , can you guys please have a look?

**Comment by mariobehling** [comment]
Thanks. I understand the reason behind this, but we have a number of points to improve in the UI. Having to take care that themes work in future increases the overhead for coming UI changes. Having themes also does not increase usability of the service. I am therefore closing it. Thank you.

**Pull request closed without merging**